app/models/import_users.rb

```diff
@@ -0,0 +1,81 @@
+class ImportUsers < Struct.new(:import, :content_path)
+  def perform
+    content_hash = JSON.parse(File.read(content_path), :symbolize_names => true)
+    
+    duplicate_users = []
+    user_for_lara = []
+    total_users_count = content_hash[:users].size
+
+    import.update_attribute(:total_imports, total_users_count)
+
+    content_hash[:users].each_with_index do |user, index|
+      new_user = User.find_by_login(user[:login]) || User.find_by_login(user[:email])
```

Comment:
shouldn't the second one be `find_by_email`?

Also I think you should only find by email. Since users can pick their login, there might be a user with a different email and the same login.  If there is an existing user with a different email and the same login then we need to keep track, so we can tell the user their login is different in the new portal. This time around it shouldn't be a problem but when this code is used in other cases it might occur more often.